src/components/QuantType.vue

```diff
@@ -45,18 +45,15 @@ export default {
       let factors = this.conversion.factors
       return math.eval(Object.keys(factors).reduce((factor, key) => {
         return factor.replace(key, factors[key])
-      }, this.conversion.expression)).toFixed(this.decimalPlaces)
-    },
-    decimalPlaces () {
-      return this.$data.conversion.decimalPlaces
+      }, this.conversion.expression)).toString()
```

Comment:
Theres a bit of inconsistency with the type of conversionFactor. Replicates constructor expects an integer, but toFixed is called on a string, maybe it could be consistent with whichever one is more appropriate

Reply:
Hey, I've taken a look at this. The way it currently is keeps it consistent with how it was before - it used toFixed(), which takes a number and outputs a string, so when I removed that I added toString() to avoid changing the return type.
The cv and standardDeviation methods also return strings, so I guess it keeps it consistent with them. 

But yes like you say, the Replicates constructor defaults it to a number, and then the adjustedAverage method uses it in a calculation like a number (crazy that this works with it as a string...)

I don't see any reason for it to be a string - I guess the reason was that the toFixed method is convenient and happens to output a string. So I could convert them all to numbers, but weirdly there doesn't seem to be an easy way to do the rounding without converting it to a string?! 

What do you think would be best?